templates/access-denied-single-job_listing.php

```diff
@@ -8,7 +8,8 @@
  * @author      Automattic
  * @package     wp-job-manager
  * @category    Template
- * @version     1.36.0
+ * @since 		1.37.0
```

Comment:
This indentation looks off

Reply:
Fixed in [a269ffc](https://github.com/Automattic/WP-Job-Manager/pull/2292/commits/a269ffc94967748db82e6dbcb5d39808ffd87288) (I just forgot to type correctly...I can fix the message by force pushing if you want)